06-building-a-program-start-to-finish/using-design-patterns/ouput/README.md

```diff
@@ -0,0 +1,8 @@
+## Output
```

Comment:
Please show code snippets of where you applied these design patterns in your work.